contributordoc.md

```diff
@@ -20,8 +20,9 @@
        * [Package linter](https://github.com/YunoHost/package_linter)
        * [Package check](https://github.com/YunoHost/package_check)
        * [Applications levels](packaging_apps_levels_fr)
-       * [App Continuous Integration](https://ci-apps.yunohost.org/jenkins/)
```

Comment:
it is not `https://ci-apps`**-dev**`.yunohost.org/jenkins/`  ?

Reply:
`https://ci-apps.yunohost.org` is our main CI. I just fixed the path which is not /jenkins anymore.